test/acceptance/kafo_configure_test.rb

```diff
@@ -173,5 +174,27 @@ module Kafo
         _(File.exist?("#{INSTALLER_HOME}/testing")).must_equal true
       end
     end
+
+    describe 'the configuration file' do
+      before do
+        @saved_values = Kafo::Configuration::DEFAULT.reject do |key, value|
+          value.nil?
+        end
+      end
+
+      it 'must only save defined values' do
+        code, _, err = run_command '../bin/kafo-configure'
+
+        _(code).must_equal 0, err
+        _(YAML.load_file(@config_file).keys.sort).must_equal @saved_values.keys.sort
+      end
+
+      it 'must not save non persisted command line options' do
+        code, _, err = run_command '../bin/kafo-configure --noop'
+
+        _(code).must_equal 0, err
+        _(YAML.load_file(@config_file).keys.sort).must_equal @saved_values.keys.sort
```

Comment:
I think you should also check that `noop` is still `false`, otherwise the test doesn't really say anything compared to the other test.

Reply:
I added a check that verifies `noop` is nil in the config file.